### PR TITLE
Add Etherpad Dockerfile

### PR DESCRIPTION
**Pull request by andrewda**
This adds an Etherpad Dockerfile based on https://github.com/ether/etherpad-lite/blob/develop/Dockerfile.

### CODE REVIEW

**Comment by ramereth** [comment]
@andrewda how do you deal with settings for each instance?

**Comment by andrewda** [comment]
@ramereth everything can be configured at run-time with environment variables. E.g. here:

https://github.com/osuosl/osl-dockerfiles/blob/fd431476ef5cc5de8d7c2b3e04bcbbbfd68ad01b/etherpad-debian/settings.json.docker#L405-L410

the `ADMIN_PASSWORD` environment variable will be used. I assume we'd just configure these before running each instance.

The only thing that needs to be configured at build time is the plugin list.

**Comment by ramereth** [comment]
@andrewda so everything is done via environment variables when starting the container?

**Comment by andrewda** [comment]
@ramereth missed your question earlier, sorry about that! Yep everything is done via environment variables when starting the container. See the [osl-app PR](https://github.com/osuosl-cookbooks/osl-app/pull/80/files#diff-1314f9303d3f75fd01a03d2be030eddbed80a7fd45e133cffae76ebde38cab73R164-R168) for details.

File: etherpad-debian/build-snowdrift.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+ETHERPAD_PLUGINS="ep_adminpads3 ep_chatdate ep_desktop_notifications ep_hide_referrer ep_small_list ep_comments_page ep_author_hover ep_webrtc"
```
Comment (ramereth) [inline etherpad-debian/build-snowdrift.sh]: I get the following error:
```
npm ERR! code ENOVERSIONS
npm ERR! No valid versions available for ep_adminpads3

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/etherpad/.npm/_logs/2020-11-02T20_16_14_988Z-debug.log
```

**Review by ramereth — CHANGES_REQUESTED** [review]

File: etherpad-debian/build-snowdrift.sh (hunk 1 above)
Comment (andrewda) [reply etherpad-debian/build-snowdrift.sh]: @ramereth try again with the latest commit, it looks like `ep_adminpads3` may have recently been moved to `ep_adminpads2` (I probably had the already downloaded `node_modules` cached)

**Review by ramereth — APPROVED** [review]

**Pull request merged**